scipy/spatial/tests/test_kdtree.py

```diff
@@ -883,6 +883,11 @@ def test_ball_point_ints():
     assert_equal(sorted([4, 8, 9, 12]),
                  sorted(tree.query_ball_point((2, 0), 1)))
 
+def test_two_big_groups():
+    points = [(1.0,)] * 100000 + [(2.0,)] * 100000
```

Comment:
I haven't tried to see if this bug is there / is fixed.  But don't do this.  On my machine it's substantially slower and the intent is less clear than the other options.  e.g.:
```
In [24]: timeit np.array([(1.0,)] * 100000 + [(2.0,)] * 100000)
10 loops, best of 3: 43.5 ms per loop

In [25]: timeit np.concatenate([np.ones((100000, 1)), np.full((100000,1), 2.0)], 0)
1000 loops, best of 3: 860 µs per loop

In [26]: timeit a = np.zeros((200000, 1)); a[100000:] = 2.0;
1000 loops, best of 3: 216 µs per loop
```